src/YAKL_atomics.h

```diff
@@ -115,72 +115,17 @@
       sycl::access::address_space::global_space>
   using relaxed_atomic_ref =
         sycl::ext::oneapi::atomic_ref< T,
-        sycl::ext::oneapi::memory_order::relaxed,
+        sycl::ext::oneapi::memory_order::seq_cst,
```

Comment:
@abagusetty , why this change?

Reply:
This was mostly for better stability for atomics for future SDK update and hardware releases in upcoming weeks. Currently this change would be no-different than previous `relaxed`